.cicdtemplate/.codemagic/README.md

```diff
@@ -0,0 +1,34 @@
+# Setup
+
+Setting up the [Codemagic](https://codemagic.io/start/) to the project starts from adding project to
+the Codemagic console.
+After that, you can follow the instruction below to setup the environment variables and deployment.
+
+# Environment Variables
+
+You can configure the environment variables for Codemagic by accessing the `Environment Variables`
+tab under `App Settings`.
+For detailed instructions on how to configure these variables, please refer
+to [this document](https://docs.codemagic.io/yaml-basic-configuration/configuring-environment-variables/#configuring-environment-variables)
+.
+
+# Deploy to Firebase App Distribution (FAD)
+
+1. Set up the Firebase SDK in your app by following
+   the [Get started guide for Android](https://firebase.google.com/docs/android/setup).
+2. Create service account for Codemagic by following
+   the [Firebase App Distribution with codemagic.yaml](https://docs.codemagic.io/yaml-publishing/firebase-app-distribution/)
+   at `Authenticating via service account` which is recommended.
+3. Update the script on `app_id` and tester `groups` in `codemagic.yaml` file:
+
+```yaml
+  publishing:
+  firebase:
+     firebase_service_account: $FIREBASE_SERVICE_ACCOUNT
+     android:
+        app_id: x:xxxxxxxxxxxx:android:xxxxxxxxxxxxxxxxxxxxxx
+        groups:
+           - androidTesters
+           - ...
+        artifact_type: 'apk' # Replace with 'aab' to only publish the Android app bundle
+```
```

Comment:
please align this with the correct indent to belong to bullet 3.

Reply:
@luongvo I tried, but the highlight color shows wrong when doing that. So I'm not sure if we should do it. 🤔

Reply:
Not sure what's wrong here 🤔 

```suggestion
   ```yaml
   publishing:
   firebase:
      firebase_service_account: $FIREBASE_SERVICE_ACCOUNT
      android:
         app_id: x:xxxxxxxxxxxx:android:xxxxxxxxxxxxxxxxxxxxxx
         groups:
            - androidTesters
            - ...
         artifact_type: 'apk' # Replace with 'aab' to only publish the Android app bundle
   ```
```

![image](https://github.com/nimblehq/android-templates/assets/16315358/8bb1a62b-9546-4818-b171-6b96245d51a5)

Reply:
@luongvo When opening with the AS:
Before change:
![Screenshot 2566-07-18 at 3 09 18 PM](https://github.com/nimblehq/android-templates/assets/12026942/29da6378-1877-437e-98c4-610460cad0a4)

After change:
![Screenshot 2566-07-18 at 3 09 10 PM](https://github.com/nimblehq/android-templates/assets/12026942/791eaf46-f126-4abd-97ba-5b42063244f0)

Reply:
@lydiasama It's the AS markdown rendering issue. Why do we rely on AS preview when the README file is almost for viewing online from the repository on GitHub?

Reply:
@luongvo I just show it for you that it's weird showing on AS. 😅  I updated it already.